app/utils/webhooks.py

```diff
@@ -452,9 +452,20 @@ async def move_message_via_webhook(  # noqa: PLR0913
     else:
         poll = message.poll
 
-    # The if expression is to skip the poll ended message if there was no poll.
-    s = _SubText(msg_data, executor, poll if message.poll is not None else None)
-    subtext = s.format() if include_move_marks else s.format_simple()
+    if executor is not None and include_move_marks and await is_moved_message(message):
+        # Append the new move mark to the existing subtext. With the exception
+        # of incredibly unlikely false positives that pass through
+        # get_moved_message() and get_moved_message_author_id(), when there is
+        # no newline in the content, the entire message content is actually the
+        # subtext, since the subtext is always present while content might not
+        # be (e.g. attachments only, sticker only, forward).
+        message.content, _, subtext = message.content.rpartition("\n")
```

Comment:
If I'm understanding correctly, this only grabs the last line of the subtext, right? I'd say it should be clarified, if so.